src/client/components/dataItem/DataItem.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import { DataItem } from './DataItem';
+
+describe('<DataItem />', () => {
+  let wrapper;
```

Comment:
I'd probably just get rid of this and just `const wrapper ...` in each of the specs.  I'd say the current implementation is technically more prone to bugs.